### DISCUSSION TITLE
How to drop the wallet and start with a fresh one?

### DISCUSSION THREAD

**Discussion by sicbitm** [lightningnetwork/lnd#7816] [General]
I have a BTCPay server with LND.

I had issues and prefer to start with a new onchain wallet. I have no open channels. How can I drop the current LND wallet and start with a fresh new one?

Thanks

**Comment by guggero** [comment]
You can simply stop `lnd`, remove the whole `.lnd` directory (wherever that is located with BTCPay), then start `lnd` again and initiate a new wallet with `lncli create`. This of course assumes that you did first send out all coins from the current wallet.

**Comment by sicbitm** [comment]
Thanks
